File: 1176/1176.py
fibonacci = [0,1]

for i in range(2,62):
    calc = fibonacci[i-1] + fibonacci[i-2]
    fibonacci.append(calc)
    
teste = int(input())

for j in range(1, teste+1):
    n = int(input())
    print(f"Fib({n}) = {fibonacci[n]}")

File: 1176/1176.php
<?php

$fibonacci = [0, 1];

for ($i = 2; $i < 62; $i++) {
    $calc = $fibonacci[$i - 1] + $fibonacci[$i - 2];
    $fibonacci[] = $calc;
}

$teste = intval(fgets(STDIN));

for ($j = 1; $j <= $teste; $j++) {
    $n = intval(fgets(STDIN));
    echo "Fib($n) = {$fibonacci[$n]}\n";
}

?>